routes/organisations/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const wizard = require('hmpo-form-wizard');
+const steps = require('./steps');
+
+module.exports = wizard(steps, null, {
```

Comment:
Just another note on pluralisation here so it doesn't get missed if we change it

Reply:
Done